virtcontainers/noop_resource_storage.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"github.com/kata-containers/runtime/virtcontainers/device/api"
+)
+
+type noopResourceStorage struct{}
```

Comment:
Nice. Since you've added this though, how about splitting `filesystem.go` into:

- `resource_storage.go` (`type resourceStorage interface` only).
- `resource_storage_filesystem.go`

And renaming `filesystem_test.go` to `resource_storage_filesystem_test.go` ?

Reply:
Yes, this makes sense, let me take care of this.

Reply:
addressed in other PR (already merged)